docs/PWADevServer.md

```diff
@@ -0,0 +1,92 @@
+# PWADevServer
+
+Utility for configuring a development OS and a `webpack-dev-server` for PWA
+development.
+
+## Usage
+
+In `webpack.config.js`:
+
+```js
+const path = require('path');
+const buildpack = require('@magento/pwa-buildpack');
+const PWADevServer = buildpack.Webpack.PWADevServer;
+
+module.exports = async env => {
+    const config {
+        /* webpack entry, output, rules, etc */
+
+        devServer: await PWADevServer.configure({
+            publicPath: '/pub/static/frontend/Vendor/theme/en_US/',
+            backendDomain: 'https://magento2.localdomain',
+            serviceWorkerFileName: 'sw.js',
+            paths: {
+                output: path.resolve(__dirname, 'web/js'),
+                assets: path.resolve(__dirname, 'web')
+            },
+            id: 'magento-venia'
```

Comment:
What does `id` mean here? Is it the name of the theme, or something else?

Reply:
It's a unique identifier for the theme. Often you're working on several simultaneous branches of a project, in different local clones. This `id` is where you'd attach a descriptive name to this body of work, like `venia-green-variant` or whatever. Right now, the tooling uses it to create the custom dev domain, e.g. `https://venia-green-variant.magento.pwadev`. In the future it could be useful in other ways.